src/Octoshift/OrganizationMigrationStatus.cs

```diff
@@ -0,0 +1,18 @@
+namespace OctoshiftCLI
+{
+    public static class OrganizationMigrationStatus
+    {
+        public const string Queued = "QUEUED";
+        public const string InProgress = "IN_PROGRESS";
+        public const string Failed = "FAILED";
+        public const string Succeeded = "SUCCEEDED";
+        public const string NotStarted = "NOT_STARTED";
+        public const string PostRepoMigration = "POST_REPO_MIGRATION";
+        public const string PreRepoMigration = "PRE_REPO_MIGRATION";
+        public const string RepoMigration = "REPO_MIGRATION";
+
+        public static bool IsSucceeded(string migrationState) => migrationState?.Trim().ToUpper() is Succeeded;
+        public static bool IsFailed(string migrationState) => migrationState?.Trim().ToUpper() is Failed;
```

Comment:
Can we flip the logic a bit to match RepositoryMigrationStatus. If the API returns a state that is not in this list, we want IsFailed() to return true.

We ran into a problem with repo migration status where it started returning a new state FAILED_VALIDATION, and we were interpreting any unexpected state as Pending, so it sent the CLI into an infinte loop.

We intentionally decided to interpret any unexpected state string as Failed.